servers/services/src/main/java/org/projectnessie/services/rest/ContentsResource.java

```diff
@@ -41,12 +42,17 @@
 @RequestScoped
 public class ContentsResource extends BaseResource implements ContentsApi {
 
+  @Context SecurityContext securityContext;
```

Comment:
any reason to not add this to the constructor and make it final?

Reply:
And make it available in `BaseResource`

Reply:
this is how it was in the original implementation that Laurent had, but it was failing with Quarkus, so I moved it out